ndk/src/hardware_buffer.rs

```diff
@@ -263,7 +273,7 @@ impl HardwareBuffer {
             None => std::ptr::null(),
         };
         construct(|res| unsafe {
-            ffi::AHardwareBuffer_lock(self.as_ptr(), usage.0, fence, rect, res)
+            ffi::AHardwareBuffer_lock(self.as_ptr(), usage.0 .0, fence, rect, res)
```

Comment:
these spaces between the `.0`s look a bit weird to me, was this intentional as `cargo fmt` doesnt seem to complain?

Reply:
@msiglreith It was actually `cargo fmt` that did this, perhaps to prevent some possible mis-interpretation of `0.0`?

We run `cargo fmt` in the CI and it's green, so it doesn't complain :)